AvalonStudio/AvalonStudio.Extensibility/Languages/ContentTypeService.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Composition;
+using System.Linq;
+
+namespace AvalonStudio.Languages
+{
+    public static class ContentTypeServiceInstance
```

Comment:
This is a small "hack" to allow classes which don't have easy access to MEF components yet to get the content type service.